libc/startup/linux/x86_64/start.cpp

```diff
@@ -76,6 +81,16 @@ void init_tls(TLSDescriptor &tls_descriptor) {
   __llvm_libc::inline_memcpy(reinterpret_cast<char *>(tlsAddr),
                              reinterpret_cast<const char *>(app.tls.address),
                              app.tls.init_size);
+  uintptr_t *stackGuardAddr = reinterpret_cast<uintptr_t *>(endPtr + 40);
```

Comment:
If this needs to be the default, then can you wrap this under a config option `LIBC_NO_STACK_PROTECTORS` so that the users can disable it if they want to.  Thanks,